docs/en/deploy/Running-Alluxio-On-Kubernetes.md

```diff
@@ -687,6 +684,136 @@ If Alluxio has already been deployed with helm and now you want to enable FUSE,
+
+Short-circuit access enables clients to perform read and write operations directly against the
+worker bypassing the networking interface.
+For performance-critical applications it is recommended to enable short-circuit operations
+against Alluxio because it can increase a client's read and write throughput when co-located with
+an Alluxio worker.
+
+#### Properties to Enable Short-Circuit Operations
+
+This feature is enabled by default, however requires extra configuration to work properly in
+Kubernetes environments.
+See sections below for how to configure short-circuit access.
+
+#### Disable Short-Circuit Operations
+To disable short-circuit operations, the operation depends on how you deploy Alluxio.
+
+> Note: As mentioned, disabling short-circuit access for Alluxio workers will result in 
+worse throughput 
+
+***Using `helm`***
+You can disable short circuit by setting the properties as below:
+
+```properties
+shortCircuit:
+  enabled: false
+```
+
+***Using `kubectl`***
+You should set the property `alluxio.user.short.circuit.enabled` to `false` in your `ALLUXIO_WORKER_JAVA_OPTS`.
+
+```properties
+-Dalluxio.user.short.circuit.enabled=false
+```
+
+You should also manually remove the volume `alluxio-domain` from `volumes` of the Pod definition 
+and `volumeMounts` of each container if existing.
+
+#### Short-Circuit Modes
+There are 2 modes for using short-circuit.
+
+##### `local`
+In this mode, the Alluxio client and local Alluxio worker recognize each other if the client hostname 
+matches the worker hostname.
+This is called *Hostname Introspection*.
+In this mode, the Alluxio client and local Alluxio worker share the tiered storage of Alluxio worker.
+
+***Using `helm`***
+You can use `local` policy by setting the properties as below:
+
+```properties
+shortCircuit:
+  enabled: true
+  policy: local
+```
+
+***Using `kubectl`***
+In your `alluxio-configmap.yaml` you should add the following properties to `ALLUXIO_WORKER_JAVA_OPTS`:
+
+```properties
+-Dalluxio.user.short.circuit.enabled=true -Dalluxio.worker.data.server.domain.socket.address= -Dalluxio.worker.data.server.domain.socket.as.uuid=false
+```
+
+##### `uuid`
+This is the **default** policy used for short-circuit.
+
+If the client or worker container is using virtual networking, their hostnames may not match.
+In such a scenario, set the following property to use filesystem inspection to enable short-circuit
+operations and **make sure the client container mounts the directory specified as the domain socket
+path**.
+Short-circuit writes are then enabled if the worker UUID is located on the client filesystem.
+
+***Domain Socket Path***
+The domain socket is a volume which should be mounted on:
+
+- All Alluxio workers
+- All application containers which intend to read/write through Alluxio
+
+This domain socket volume refers to a `PersistentVolumeClaim` by its name.
+You should provision to the `PersistenceVolumeClaim` with a `PersistentVolume` of kind being either
```

Comment:
this sentence doesn't make sense to me

Reply:
I tried to rephrase it. PTAL at the latest version. Thanks!